Makefile

```diff
@@ -23,7 +23,7 @@ install: ## Install for the current user using the default python command
 	python3 setup.py build_ext --inplace && python setup.py install --user
 
 test: ## Run unit tests
-	pytest --cov-report html --cov=sktime --showlocals --durations=20 --pyargs $(PACKAGE)
+	pytest --cov-report html --cov=sktime --showlocals --durations=20 -n 4 --pyargs $(PACKAGE)
```

Comment:
Do we need to specify pytest-xdist somewhere to use multiple processes and the `-n` flag here?